pymc/tests/test_sampling.py

```diff
@@ -1212,3 +1212,58 @@ def test_sample_deterministic():
         idata = pm.sample(chains=1, draws=50, compute_convergence_checks=False)
 
     np.testing.assert_allclose(idata.posterior["y"], idata.posterior["x"] + 100)
+
+
+class TestDraw(SeededTest):
```

Comment:
In these tests it would be good to check that the draws are not all the same, which would happen by default if we didn't do some magic in the `Model`

Reply:
Hi sure, this test will be added as well.

Reply:
Hi, what is the magic in the` Model`? I tried to draw two same samples by using seed with `rng_seeder` , but not success.

```
with pm.Model(**rng_seeder**=np.random.seed(123)):
    y = pm.Normal("y")

y_draws_1 = pm.draw(y, 100) 
y_draws_2 = pm.draw(y, 100)
# Check if the draw function will draw same samples using rng_seeder in pm.Model
npt.assert_array_equal(y_draws_1, y_draws_2) # This failed :(
```

Reply:
It's good that they are different, that means the "magic" is working.
If you did the following calls outside of a model they would yield the same draws:

```python
# no model
y = pm.Normal.dist()

y_draws_1 = pm.draw(y, 100) 
y_draws_2 = pm.draw(y, 100)
```

Reply:
Hi, that's great. Thanks @ricardoV94  :+1: